src/Servers/Kestrel/Core/src/Internal/Infrastructure/IKestrelTrace.cs

```diff
@@ -72,7 +72,7 @@ internal interface IKestrelTrace : ILogger
 
         void HPackDecodingError(string connectionId, int streamId, HPackDecodingException ex);
 
-        void HPackEncodingError(string connectionId, int streamId, HPackEncodingException ex);
+        void HPackEncodingError(string connectionId, int streamId, Exception ex);
```

Comment:
If `HPackEncodingError` takes `Exception` then should `HPackDecodingError` (and QPack equivalent) be updated to match? I like the consistency.